clarity/tests/arkadiko-freddie-v1-1_test.ts

```diff
@@ -79,7 +79,7 @@ Clarinet.test({
     result.expectOk().expectUint(1925000);
 
     let call = vaultManager.getCurrentCollateralToDebtRatio(1, deployer);
-    call.result.expectOk().expectUint(200);
+    call.result.expectOk().expectUint(199);
```

Comment:
REQ/Q: this might be 199 since the stability fee is still commented out (see TODO in `calculate-current-collateral-to-debt-ratio`).. but probably 200 is correct, not sure

Reply:
Yes I think so, but I have no clue how to fix the TODO in Freddie :/